src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/HtmlTag.java

```diff
@@ -434,57 +434,76 @@ public enum Flag {
 
     public enum Attr {
         ABBR,
+        ACCESSKEY(true),
         ALIGN,
         ALINK,
         ALT,
-        ARIA_ACTIVEDESCENDANT,
-        ARIA_CONTROLS,
-        ARIA_DESCRIBEDBY,
-        ARIA_EXPANDED,
-        ARIA_LABEL,
-        ARIA_LABELLEDBY,
-        ARIA_LEVEL,
-        ARIA_MULTISELECTABLE,
-        ARIA_OWNS,
-        ARIA_POSINSET,
-        ARIA_SETSIZE,
-        ARIA_READONLY,
-        ARIA_REQUIRED,
-        ARIA_SELECTED,
-        ARIA_SORT,
+        ARIA_ACTIVEDESCENDANT(true),
```

Comment:
The enum `Attr` holds two values now, the second representing whether it's a global attribute.